src/components/Card/components/Section/Section.tsx

```diff
@@ -27,8 +21,21 @@ export default function Section({children, title, subdued, fullWidth}: Props) {
 
   return (
     <div className={className}>
-      {headerContent}
+      {renderHeaderContent()}
```

Comment:
As a general pattern we don't usually create functions and have them return markup. Instead we usually create a property in the render function and assign it. (E.g)

```tsx
render()
...
   const headerMarkup = title && (
    typeof title === 'string'
      ? <Subheading>{title}</Subheading>
      : title
   )
...
```

Reply:
I went with a function for more readability, I don't really find nested ternary operators readable.
I can change this though to follow our style guide